internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -171,6 +172,22 @@ func isContainerRequestWholeCPU(c *oci.Container) bool {
 	return *(c.Spec().Linux.Resources.CPU.Shares)%1024 == 0
 }
 
+func setCPUSLoadBalancingWithRetry(ctx context.Context, c *oci.Container, enable bool) error {
+	log.Infof(ctx, "Disable cpu load balancing for container %q", c.ID())
+	// it is possible to have errors during reading or writing to sched_domain files because
+	// that kernel rebuilds it with updated values
+	// the retry will not fix it for 100% but should reduce the possibility for failures to minimum
+	// TODO: re-visit once we will have some more acceptable cgroups hierarchy to disable CPU load balancing
+	// correctly via cgroups, see -https://bugzilla.redhat.com/show_bug.cgi?id=1946801
+	return wait.PollImmediate(time.Second, 5*time.Second, func() (bool, error) {
+		if err := setCPUSLoadBalancing(c, enable, schedDomainDir); err != nil {
```

Comment:
should we only do this if `os.IsNotExist(err)`?

Reply:
yes it makes sense

Reply:
Done